src/Lykke.Service.Nem.Api/Startup.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using JetBrains.Annotations;
+using Lykke.Sdk;
+using Lykke.Service.BlockchainApi.Contract;
+using Lykke.Service.BlockchainApi.Sdk;
+using Lykke.Service.BlockchainApi.Sdk.Domain.DepositWallets;
+using Lykke.Service.Nem.Api.Services;
+using Lykke.Service.Nem.Api.Settings;
+using Lykke.SettingsReader;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Lykke.Service.Nem.Api
+{
+    [UsedImplicitly]
+    public class Startup
+    {
+        private readonly LykkeSwaggerOptions _swaggerOptions = new LykkeSwaggerOptions
+        {
+            ApiTitle = "Nem Api",
+            ApiVersion = "v1"
+        };
+
+        [UsedImplicitly]
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            return services.BuildServiceProvider<AppSettings>(options =>
+            {
+                options.SwaggerOptions = _swaggerOptions;
+
+                options.Logs = logs =>
+                {
+                    logs.AzureTableName = "NemApiLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.NemApi.Db.LogsConnString;
```

Comment:
Add custom Slack channels bcn and bcn-imprtnt. disable spam guarding by channelOptions.SpamGuard.DisableGuarding(); and enable health notifications by channelOptions.IncludeHealthNotifications();

Reply:
Fixed except health notification